ruff.spec

```diff
@@ -0,0 +1,8 @@
+### RPM external ruff 0.5.6
+Source: https://github.com/astral-sh/ruff/releases/download/%{realversion}/ruff-x86_64-unknown-linux-gnu.tar.gz
+
+%setup
+tar -zxvf ruff-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
you need something like
```
%prep
%setup -n ruff-x86_64-unknown-linux-gnu
```

Reply:
And %setup would already untar and move to %{i}/bin?

Reply:
no, it only unpack the source ( please read rpm docs :-) )